File: Difficulty: Easy/Move All Zeroes to End/move-all-zeroes-to-end.java
class Solution {
    void pushZerosToEnd(int[] arr) {
        // code here
        int n = arr.length;
        int i = 0;
        int k = 0;
        while(i < n){
            if(arr[i] != 0){
                arr[k++] = arr[i];
            }
            i++;
        }
        
        while(k<n){
            arr[k++] = 0;
        }
        
    }
}

File: Difficulty: Easy/Move All Zeroes to End/move-all-zeroes-to-end.py
class Solution:
	def pushZerosToEnd(self, arr):
    	# code here
    	zeros = 0
    	idx = 0
    	for i in arr:
    	    if i != 0:
    	        arr[idx] = i
    	        idx += 1
    	
    	
    	while idx < len(arr):
    	    arr[idx] = 0
    	    idx += 1
    	
    	
    	return arr